run

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env sh
 
+SCRIPT_PATH=$(readlink -f "$0")
+PROJECT_DIR=$(dirname "$SCRIPT_PATH")
+
 # use the script generated by Gradle (does all the relative-path-magic for the dependencies)
-build_dir/install/minijavac/bin/minijavac "$@"
+$PROJECT_DIR/build_dir/install/minijavac/bin/minijavac "$@"
```

Comment:
Please don't forget the quotes around the path (`"$PROJECT_DIR/build_dir/install/minijavac/bin/minijavac"`)
in case somebody uses whitespace in there directory names (shame on them!) :wink: